src/settings/PatronNotices/PatronNoticeDetail.js

```diff
@@ -114,18 +118,21 @@ class PatronNoticeDetail extends React.Component {
         >
           <Accordion
             id="email-template-detail"
-            label={<FormattedMessage id="ui-circulation.settings.patronNotices.email" />}
+            label={formatMessage({ id: 'ui-circulation.settings.patronNotices.email' })}
           >
             { emailTemplate &&
-              <>
+              <div data-testid="emailAccordionContent">
                 <Row>
                   <Col xs={8}>
                     <KeyValue
-                      label={<FormattedMessage id="ui-circulation.settings.patronNotices.subject" />}
+                      label={formatMessage({ id: 'ui-circulation.settings.patronNotices.subject' })}
                       value={notice.localizedTemplates.en.header}
                     />
                   </Col>
-                  <Col xs={4}>
+                  <Col
+                    data-testid="previewButtonColumn"
+                    xs={4}
+                  >
                     <Button onClick={this.openPreviewDialog}>
                       <FormattedMessage id="ui-circulation.settings.patronNotices.preview" />
```

Comment:
Can we here replace `FormattedMessage` by `formatMessage` too, and remove `FormattedMessage` from imports as unnecessary?